test/tests/melt_pool_mass/tests

```diff
@@ -3,6 +3,7 @@
   [mass]
     type = 'Exodiff'
     input = 'mass.i'
+    abs_zero = 6e-10   # Adjusted due to failure on Apple Silicon systems
```

Comment:
Had you tried tightening the tolerances in the input file before loosening this tolerance to avoid the apple silicon system failure?

Reply:
I did. Unfortunately it didn't have much impact at all. I could tweak solver tolerances and settings further, definitely, if you want me to do so! Just won't happen for another couple of weeks as I'm about to go on travel again.

Reply:
That's fine as is. I wanted to check, and document, that you had tried with the input file tolerances before changing the exodiff

Reply:
Sounds good - as you can probably tell from setting the absolute zero, the diffing was very very small on equally small variable values...that's likely why the tolerance adjustments weren't very effective.